extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -264,20 +264,46 @@ export class SchemaCompareResult {
 		return data;
 	}
 
+	private getFormattedScript(diffEntry: azdata.DiffEntry, getSourceScript: boolean): string {
+		let script = '';
+		if (getSourceScript) {
+			script = diffEntry.sourceScript === null ? '\n' : this.getAggregatedScript(diffEntry, getSourceScript);
+		} else {
+			script = diffEntry.targetScript === null ? '\n' : this.getAggregatedScript(diffEntry, getSourceScript);
+		}
+
+		// add new lines after 'GO's
+		script = script.replace(new RegExp('GO', 'g'), 'GO\n');
+
```

Comment:
This is going to replace ALL GO's right? So if I have

SELECT 'GO' FROM sys.tables

then it's going to return

SELECT 'GO\n' FROM sys.tables

which we shouldn't be doing. 

Why are you adding newlines after the GO's here anyways?

Reply:
I was adding newlines after the GOs because that's what SSDT does